lib/BridgedRoom.js

```diff
@@ -238,6 +238,79 @@ BridgedRoom.prototype.onSlackMessage = function(message) {
     });
 };
 
+
+// These functions are copied and modified from the Gitter AS
+// idx counts backwards from the end of the string; 0 is final character
+function rcharAt(s,idx) { return s.charAt(s.length-1 - idx); }
```

Comment:
These feel like things that deserve their own utility class, rather than being left in BridgedRoom?